db/projects/komodo/project.yaml

```diff
@@ -0,0 +1,90 @@
+name: Komodo
+web:
+  - 'https://komodoplatform.com/latest-news/'
+start_date: '2014-01-01'
+description: 'Open-Source Infrastructure For An Interoperable Blockchain Ecosystem'
+ledgers:
+  - id: komodo
+    name: Komodo
+    type: blockchain
+    networks:
+      - id: main
+        algorithm: Equihash
+        proof_type: dPoW/PoW
```

Comment:
I think here we can still make some lists to be able to sort it in the future. It would be really hard to search if sometimes would be used as dPoW/PoW, sometimes dPoW+PoW, dPoW + PoW, Pow/dPoW...

Reply:
Will be done by trello ticket (on mario)